src/main/java/moze_intel/projecte/network/packets/UpdateWindowLongPKT.java

```diff
@@ -9,16 +9,16 @@
 import net.minecraftforge.fml.common.network.simpleimpl.MessageContext;
 
 // Version of SPacketWindowProperty that does not truncate the `value` arg to a short
-public class UpdateWindowIntPKT implements IMessage
+public class UpdateWindowLongPKT implements IMessage
```

Comment:
don't rename this. make a new one for long and only use it when necessary

I don't want to waste 4 more bytes per update for fields that don't need it